### PR TITLE
SEO Work: JS, Python, Git

### PR DESCRIPTION
**Pull request by Dusch4593**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/built-in-functions/terms/input/input.md [hunk 1]
```diff
@@ -1,45 +1,54 @@
 ---
 Title: 'input()'
-Description: 'Takes in a value from the user and converts the value into a string.'
+Description: 'Prompts the user for data and returns as a string.'
```
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/input/input.md:3]:
```suggestion
Description: 'Prompts the user for data and returns it as a string.'
```

File: content/python/concepts/keywords/terms/global/global.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+Title: 'global'
+Description: 'Creates a global variable that can be updated '
```
Comment (SSwiniarski) [inline content/python/concepts/keywords/terms/global/global.md:3]:
```suggestion
Description: 'Creates a global variable that can be updated.'
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Dusch4593, reviewed and had only a couple of minor changes.

**Comment by Dusch4593** [comment]
@SSwiniarski Implemented your suggestions! 😄

**Review by SSwiniarski — APPROVED** [review]
Looks good for another review.

File: content/javascript/concepts/arrays/terms/splice/splice.md [hunk 3]
```diff
@@ -6,6 +6,7 @@ Subjects:
   - 'Computer Science'
```
Comment (KTom101) [inline content/javascript/concepts/arrays/terms/splice/splice.md:6]: Update description to say "Modifies an array"

**Review by KTom101 — CHANGES_REQUESTED** [review]
@Dusch4593 I added some changes to this entry.

File: content/javascript/concepts/arrays/terms/sort/sort.md [hunk 4]
```diff
@@ -1,61 +1,70 @@
...
-array.sort((element1, element2) => element1 - element2);
-```
+An optional function is used to define how items are sorted. This is done by iterating over the `array` and comparing every `firstElement` and `secondElement` in the `/* compison body */`.
+
+## Example
 
-## Examples
+In the following example, the `.sort()` method is applied to two arrays, `letters` and `numbers` (a mix of floats and integers):
 
 Sort strings in an array alphabetically:
```
[the remark this answers is not in the document]
Comment (Dusch4593) [reply content/javascript/concepts/arrays/terms/sort/sort.md:39]: Sorry, I meant to remove that 😂

**Comment by Dusch4593** [comment]
@KTom101 Finished implementing your feedback 😄

**Review by KTom101 — APPROVED** [review]
👍

**Pull request merged**